src/Microsoft.Health.Fhir.Liquid.Converter/Processors/BaseProcessor.cs

```diff
@@ -17,14 +17,13 @@ namespace Microsoft.Health.Fhir.Liquid.Converter.Processors
 {
     public abstract class BaseProcessor : IFhirConverter
     {
-        private readonly ProcessorSettings _settings;
-        private const int _maxIterations = 100000;
-
         protected BaseProcessor(ProcessorSettings processorSettings = null)
         {
-            _settings = processorSettings;
+            Settings = processorSettings ?? new ProcessorSettings();
```

Comment:
how about changing the setting to required ?

Reply:
Updated the ```processorSettings``` as required